Formula/z/zbctl.rb

```diff
@@ -1,6 +1,6 @@
 class Zbctl < Formula
   desc "Zeebe CLI client"
-  homepage "https://docs.camunda.io/docs/apis-clients/cli-client/index/"
+  homepage "https://docs.camunda.io/docs/apis-tools/community-clients/go-client/"
```

Comment:
This seems to be the one:

```suggestion
  homepage "https://docs.camunda.io/docs/apis-tools/community-clients/cli-client/"
```